app/views/liquids/_stackexchange.html.erb

```diff
@@ -1,14 +1,14 @@
 <div class="ltag__stackexchange--container">
   <div class="ltag__stackexchange--title-container">
-    <header>
+    <header id="ltag-stackexchange-header">
       <div class="ltag__stackexchange--title">
       <% logo_to_use = site == "stackoverflow" ? "stackoverflow-logo.svg" : "stackexchange-logo.svg" %>
-        <h1>
-          <img src="<%= ActionController::Base.helpers.asset_path(logo_to_use) %>" alt="">
-            <a href="<%= post_url %>" target="_blank" rel="noopener noreferrer">
-              <%= post_type == "question" ? CGI.unescapeHTML(title) : t("views.liquids.stackexchange.answer.text_html", answer: tag.span(t("views.liquids.stackexchange.answer.answer"), class: "title-flare"), title: CGI.unescapeHTML(title)) %>
-            </a>
-        </h1>
+        <div class="ltag__stackexchange--header">
+          <img src="<%= ActionController::Base.helpers.asset_path(logo_to_use) %>" alt="<% site %> logo">
```

Comment:
```suggestion
          <img src="<%= ActionController::Base.helpers.asset_path(logo_to_use) %>" alt="">
```
Let's keep this empty - empty `alt` attributes indicate that an image is purely decorative, allowing screen reader users to pass over them without additional noise.